webui/react/src/services/apiConfig.ts

```diff
@@ -1022,11 +1024,7 @@ export const getTrialWorkloads: DetApi<
       WorkloadFilterParamMap[params.filter || 'FILTER_OPTION_UNSPECIFIED'] ||
         'FILTER_OPTION_UNSPECIFIED',
       undefined,
-      params.metricType
-        ? params.metricType === Type.MetricType.Training
-          ? 'METRIC_TYPE_TRAINING'
-          : 'METRIC_TYPE_VALIDATION'
-        : undefined,
+      undefined, // Specifically set to `undefined` to return custom group metrics.
```

Comment:
This has an effect on the Workloads table, where "Has Validation" or "Has Checkpoint or Validation" appears to include custom metric groups rows.